frontends/election-manager/src/app.test.tsx

```diff
@@ -602,17 +636,9 @@ test('tabulating CVRs', async () => {
 
   fireEvent.click(getByText('Close'));
 
-  // Check that a zero report can be generated on the L&A tab even after CVRs have been tabulated
+  // Check L&A Materials are unavailable after live CVRs have been loaded
   fireEvent.click(getByText('L&A'));
-  fireEvent.click(
-    getByText('Print the pre-election Unofficial Full Election Tally Report')
-  );
-  await findByText('Printing');
-  expect(printer.print).toHaveBeenCalledTimes(1);
-  expect(mockKiosk.log).toHaveBeenCalledWith(
-    expect.stringContaining(LogEventId.TallyReportPrinted)
-  );
-  expect(getAllByText('0').length).toBe(40);
+  expect(queryByText(zeroReportTitle)).toBeNull();
```

Comment:
Might additionally want to check for the `L&amp;A materials are not available after live CVRs have been imported.` text